doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -53,27 +56,46 @@ lock.
 
 =item *
 
-CRYPTO_THREAD_read_lock() locks the provided B<lock> for reading.
+CRYPTO_THREAD_read_lock() locks the provided I<lock> for reading.
 
 =item *
 
-CRYPTO_THREAD_write_lock() locks the provided B<lock> for writing.
+CRYPTO_THREAD_write_lock() locks the provided I<lock> for writing.
 
 =item *
 
-CRYPTO_THREAD_unlock() unlocks the previously locked B<lock>.
+CRYPTO_THREAD_unlock() unlocks the previously locked I<lock>.
 
 =item *
 
-CRYPTO_THREAD_lock_free() frees the provided B<lock>.
+CRYPTO_THREAD_lock_free() frees the provided I<lock>.
 
 =item *
 
-CRYPTO_atomic_add() atomically adds B<amount> to B<val> and returns the
-result of the operation in B<ret>. B<lock> will be locked, unless atomic
+CRYPTO_atomic_add() atomically adds I<amount> to I<*val> and returns the
+result of the operation in I<*ret>. I<lock> will be locked, unless atomic
 operations are supported on the specific platform. Because of this, if a
 variable is modified by CRYPTO_atomic_add() then CRYPTO_atomic_add() must
-be the only way that the variable is modified.
+be the only way that the variable is modified. If atomic operations are not
+supported and I<lock> is NULL, then the function will fail.
+
+=item *
+
+CRYPTO_atomic_or() atomically ors I<op> to I<*val> and returns the
```

Comment:
Is it a boolean/logical or?   And not a C "any non-zero bit is true" kind of or?  Like ```|``` versus ```||```?  I'm sure it is, but it might be worth  calling that out in the docs here and consider changing the name.

Reply:
Its a bitwise or (i.e. `|` not `||`). I have made this clearer in the docs. I don't think its necessary to change the name - the underlying atomic operation names don't feel the need to make this distinction.

Reply:
> made it clearer in the docs.

WFM, thanks.